lib/repl.js

```diff
@@ -928,6 +951,16 @@ function complete(line, callback) {
   function completionGroupsLoaded(err) {
     if (err) throw err;
 
+    if (warning.type) {
+      self._writeToOutput('\r\n');
+      process.emitWarning(
+        warning.message,
+        warning.type,
+        undefined,
+        undefined,
+        true);
```

Comment:
Ugh... forgot about that change. It wasn't added to the docs and wasn't very keen on it in the first place. In this case, why does the warning need to be emitted immediately?